code/modules/projectiles/projectile/bullets/revolver.dm

```diff
@@ -1,8 +1,16 @@
-// 7.62x38mmR (Nagant Revolver)
+// .310 Short (Sakhno-Manni Revolver)
 
-/obj/projectile/bullet/n762
-	name = "7.62x38mmR bullet"
-	damage = 60
+/obj/projectile/bullet/short310
+	name = ".310 Short bullet"
+	damage = 35
+	wound_bonus = -45
+	wound_falloff_tile = 0
+	speed = 1.3
+
+/obj/projectile/bullet/short310/surplus
+	name = ".310 Short surplus bullet"
+	damage = 30
```

Comment:
No need to reduce the damage. ``weak_to_armour`` basically does the job for you in almost any relevant instance. The intent with surplus/weak_to_armour was to make a weapon/bullet mob/armorless opponent focused, rather than outright weaker than baseline.